scripts/lint-commits.sh

```diff
@@ -14,7 +14,14 @@ if [[ -n "${JENKINS_URL}" ]]; then
 
     # Lint PR title
     if [[ -n ${CHANGE_TITLE} ]]; then
+      titleLength=$(echo -n "${CHANGE_TITLE}" | wc -c)
+      titleResult=0
+      if [ "${titleLength}" -ge 75 ] ; then
```

Comment:
Header must not be longer than 72 characters, so PRs with 65 chars and 8 chars (` (#ABCD)`) then should fail

```suggestion
      if [ "${titleLength}" -ge 65 ] ; then
```